### PR TITLE
Improve scaling in Rayleigh quotient example

### PR DESCRIPTION
**Pull request by mateuszbaran**
I'm not sure if this issue should be discussed in the example or not.

### CODE REVIEW

**Comment by mateuszbaran** [comment]
Hm, the Documenter error isn't even in the notebook I've modified.

**Comment by kellertuer** [comment]
I think it is ok without mentioning that in the docs, thanks for fixing that.

**Comment by kellertuer** [comment]
For the error, I will try to fix that soon-ish; just that this week was a bit crazy (in my head it is something like Monday evening still, which means there was a bit too much of work this week).

**Comment by mateuszbaran** [comment]
You're welcome :slightly_smiling_face: . This PR can wait until you have more time.

File: examples/RayleighQuotient.qmd [hunk 1]
```diff
@@ -162,7 +162,7 @@ We can also benchmark both
 @benchmark gradient_descent($M, $f, $grad_f, $p0)
 ```
 
-We see, that the conversion costs a bit of performance, but if the Euclidean gradient is easier to compute, this might still be ok.
+We see, that the conversion costs a bit of performance, but if the Euclidean gradient is easier to compute, this might still be OK.
```
Comment (kellertuer) [inline examples/RayleighQuotient.qmd:165]: Why do you want to capitalise OK? I think “ok” is also ok. Maybe it is overall a bit informal so we could maybe rephrase the final part of the sentence anyways.

Comment (mateuszbaran) [reply examples/RayleighQuotient.qmd:165]: Capitalized OK is more formal: https://www.writing-skills.com/ok-ok-okay-how-do-you-write-ok , so I prefer to use it in technical writing (like here).

Comment (mateuszbaran) [reply examples/RayleighQuotient.qmd:165]: oKaY :wink:

Comment (kellertuer) [reply examples/RayleighQuotient.qmd]: Rephrased it.

**Comment by kellertuer** [comment]
I deleted all caches here and checked again locally – it runs fine there.

**Comment by kellertuer** [comment]
Something is super strange here. Documenter fails with more and more messages that get more and more strange. And locally on my machine it just runs fine even when deleting all caches (`_freeze`) locally. I am running a bit out of ideas what documenter might be doing on CI that is different from my machine. But well – then this PR has to wait a bit still until I get new ideas.

**Comment by mateuszbaran** [comment]
It's weird... `ArmijoLinesearch` doesn't initialize a reasonable `candidate_point` if you don't give it the correct manifold, so I'm not sure how it could ever work.

**Comment by mateuszbaran** [comment]
A quick fix would be to just give the right manifold to `ArmijoLinesearch` constructor.

**Comment by kellertuer** [comment]
That did indeed resolve the issue. I think we missed that the originally nonbreaking update of ArmijoLinesearch now has a field to store the candidate and that does require the manifold (or specifying that field).

Whatever. Still super surprising that it ran just fine on my machine but not on CI.
But a positive point is that the docs here are now rendered on Julia 1.10.

**Review by kellertuer — APPROVED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ please refactor the code, such that the gradient, or other function is in the co
 `src/functions/` and follows the naming scheme:
 
 * cost functions are always of the form `cost_` and a fitting name
-* gradient functions are always of the the `gradient_` and a fitting name, followed by an `!`
+* gradient functions are always of the`gradient_` and a fitting name, followed by an `!`
```
Comment (mateuszbaran) [inline CONTRIBUTING.md:46]:
```suggestion
* gradient functions are always of the `gradient_` and a fitting name, followed by an `!`
```

File: examples/Difference-of-Convex-Rosenbrock.qmd [hunk 3]
```diff
@@ -13,7 +13,7 @@ This example is the code that produces the results in [BergmannFerreiraSantosSou
 Both the Rosenbrock problem
 
 ```math
-    \operatorname*{argmin}_{x\in\mathbb R^2} a\bigl( x_1^2-x_2\bigr)^2 + \bigl(x_1-b\bigr)^2,
+    \operatorname*{argmin}_{x\inℝ^2} a\bigl( x_1^2-x_2\bigr)^2 + \bigl(x_1-b\bigr)^2,
```
Comment (mateuszbaran) [inline examples/Difference-of-Convex-Rosenbrock.qmd:16]:
```suggestion
    \operatorname*{argmin}_{x\in ℝ^2} a\bigl( x_1^2-x_2\bigr)^2 + \bigl(x_1-b\bigr)^2,
```

File: examples/Total-Variation.qmd [hunk 4]
```diff
@@ -41,9 +41,9 @@ where $α > 0$ is a weight parameter.
 
 The challenge here is that most classical algorithm, like gradient descent or Quasi Newton, assume the cost $f(p)$ to be smooth such that the gradient exists at every point. In our setting that is not the case since the distacen is not differentiable for any $p_i=p_{i+1}$. So we have to use another technique.
 
-## THe Cyclic Proximal Point algorithm
+## The Cyclic Proximal Point algorithm
 
-If the cost consists of a sum of functions, where each of the proximal maps is “easy to evaluate”, for best of cases in closed form, we can “apply the proximal maps in a cyclic fashion” and optain the the [Cyclic Proximal Point Algorithm](https://manoptjl.org/stable/solvers/cyclic_proximal_point/) [Bacak:2014](@cite).
+If the cost consists of a sum of functions, where each of the proximal maps is “easy to evaluate”, for best of cases in closed form, we can “apply the proximal maps in a cyclic fashion” and optain the[Cyclic Proximal Point Algorithm](https://manoptjl.org/stable/solvers/cyclic_proximal_point/) [Bacak:2014](@cite).
```
Comment (mateuszbaran) [inline examples/Total-Variation.qmd:46]:
```suggestion
If the cost consists of a sum of functions, where each of the proximal maps is “easy to evaluate”, for best of cases in closed form, we can “apply the proximal maps in a cyclic fashion” and optain the [Cyclic Proximal Point Algorithm](https://manoptjl.org/stable/solvers/cyclic_proximal_point/) [Bacak:2014](@cite).
```
